pkg/querier/worker/worker.go

```diff
@@ -28,13 +28,14 @@ type Config struct {
 	FrontendAddress       string        `yaml:"frontend_address"`
 	SchedulerAddress      string        `yaml:"scheduler_address"`
 	DNSLookupPeriod       time.Duration `yaml:"dns_lookup_duration" category:"advanced"`
-	MaxConcurrentRequests int           `yaml:"-"` // Must be same as passed to PromQL Engine.
+	MaxConcurrentRequests int           `yaml:"max-concurrent"`
```

Comment:
```suggestion
	MaxConcurrentRequests int           `yaml:"max_concurrent"`
```

I was reviewing open PRs for changes that will change existing configuration options and I came across this. The convention with the rest of our flags is to use snake_case, can you change this PR to reflect that?

Reply:
Updated.
Thanks for your review!  💚 💙 💜 💛 ❤️